tools/OpenConsole.psm1

```diff
@@ -41,7 +41,7 @@ function Import-LocalModule
         Write-Verbose "$Name already downloaded"
         $versions = Get-ChildItem "$modules_root\$Name" | Sort-Object
 
-        Get-ChildItem -Path "$modules_root\$Name\$($versions[0])\$Name.psd1" | Import-Module
+        Get-ChildItem -Path "$modules_root\$Name\$($versions[0].name)\$Name.psd1" | Import-Module

```

Comment:
I wonder-- can we do `$($versions[0].FullName)\$Name.psd1` and have it work right in both places?

Reply:
yup.